data/json/items/comestibles.json

```diff
@@ -6738,6 +6759,30 @@
         "volume" : 0,
         "charges" : 4,
         "fun" : 1
+    },{
+        "type" : "COMESTIBLE",
+        "id" : "dry_mushroom",
+        "name" : "dried hallucinogenic mushroom",
+        "weight" : 15,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 0,
+        "use_action" : "FUN_HALLU",
+        "stim" : 0,
+        "container" : "null",
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : -1,
+        "healthy" : 0,
+        "addiction_potential" : 0,
+        "nutrition" : 8,
+        "description" : "Some hallucinogenic mushrooms which have been dehydrated for storage.  Will still cause hallucinations if eaten.",
```

Comment:
Either make the descriptions be about the object in singular, or change the base name. 
``dried hallucinogenic mushrooms`` 
or
``A hallucinogenic mushroom which...``
I'd say go with option 2 as most items have their description in singular